src/components/shell/shell.js

```diff
@@ -276,6 +277,7 @@ class Shell extends BaseComponent {
     if (this.props.metadataLoaded) playerClasses.push(style['state-' + this.props.currentState]);
     if (this.props.seekbarDraggingActive) playerClasses.push(style.hover);
     if (this.props.playerClientRect && this.props.playerClientRect.width <= 480) playerClasses.push(style.sizeSm);
+    if (this.props.fullscreen && this.player.env.os.name === 'iOS') playerClasses.push(style.iOSFullscreen);
```

Comment:
Let's move this condition from between the size condition (row above and below).
Also, let's just add fullscreen state class, e.g. `if (this.props.fullscreen) playerClass.push(style.inFullscreen)` (or maybe just fullscreen)
and add user agent class, so:
```
let playerClasses = [style.player, style.skinDefault, this.player.env.os.name, this.player.env.browser.name];
playerClasses.push(props.playerClasses);
```

Probably need to add the `playkit` prefix to these new classes.

ref in V2: https://github.com/kaltura/mwEmbed/blob/ba045a91abdee7d0761ba543f5068b4335c04c9f/modules/EmbedPlayer/resources/mw.PlayerLayoutBuilder.js#L152-L177